PINCache/PINCache.h

```diff
@@ -101,7 +120,7 @@ typedef void (^PINCacheObjectBlock)(PINCache *cache, NSString *key, id __nullabl
  @param rootPath The path of the cache on disk.
  @result A new cache with the specified name.
  */
-- (instancetype)initWithName:(NSString *)name rootPath:(NSString *)rootPath NS_DESIGNATED_INITIALIZER;
+- (instancetype)initForExtensionsWithName:(NSString *)name rootPath:(NSString *)rootPath NS_DESIGNATED_INITIALIZER;
```

Comment:
This will look pretty weird in Swift, `init(forExtensionsWithName:, rootPath:)`, seems like you're giving the name of the extensions that it's being initialized for. I don't have a great suggestion, I'll think on it...